source/DataFile.cpp

```diff
@@ -231,4 +233,13 @@ void DataFile::LoadData(const string &data)
 		if(mixedIndentation)
 			node.PrintTrace("Warning: Mixed whitespace usage at line");
 	}
+
+	root.SetStore(store);
+}
+
+
+
+std::shared_ptr<ConditionsStore> DataFile::Store()
```

Comment:
```suggestion
std::shared_ptr<ConditionsStore> DataFile::Store() const
```

shouldnt this be const?

Reply:
No. The conditions need a non-const store so they can assign values.

I could add a const version too, but it isn't needed for the Condition functionality.

Reply:
fine then

Reply:
It's there now.

Reply:
just resolve the converstions then 👌